terraform/modules/paas/main.tf

```diff
@@ -155,6 +155,18 @@ resource "cloudfoundry_service_instance" "postgres" {
   }
 }
 
+resource "cloudfoundry_service_instance" "postgres_snapshot" {
+  count        = var.app_environment == "prod" || var.app_environment == "qa" ? 1 : 0
```

Comment:
Could we use a new boolean variable such as "create_snapshot_database"? This may be cleaner than hardcoding the environment names here